README.md

```diff
@@ -20,7 +20,11 @@ yarn add @arcadia-eng/connect-react
 
 # Quick Start
 
-A Connect Token is required to instantiate Connect. See [Creating Connect Tokens](https://developers.arcadia.com/#section/Authentication/Connect) for instructions. Note that a general Access Token will not work for this purpose.
+A Connect Token is required to instantiate Connect. See [Creating Connect Tokens](https://developers.arcadia.com/#section/Authentication/Connect) for instructions. Keep in mind the following:
+
+- A general Access Token will not work to instantiate Connect
+- Using sandbox credentials to create a Connect Token will create resources in the sandbox environment
+- Using production credentials to create a Connect Token will create resources in the production environment
```

Comment:
Not sure -- but maybe we need to align on calling these keys? Is that how the dashboard is functioning @BrandonGoren

Reply:
Language is more of a @johnnyman727 question - we call these `tokens` in the API.

Reply:
We should call the API Client ID "keys" in our documentation to better differentiate from utility credentials.

Reply:
updated 👍